src/Components/Alert/Hooks/__tests__/Utils/AlertProviderTestPage.tsx

```diff
@@ -0,0 +1,39 @@
+import { RootTestPage } from "DevTools/RootTestPage"
+
+export class AlertProviderTestPage extends RootTestPage {
```

Comment:
I feel bad flagging this, because it seems fine, but yeah, this is a deprecated pattern